Base/usr/share/man/man1/FontEditor.md

```diff
@@ -69,7 +71,7 @@ The main workspace has three partitions:
 Metadata and unicode blocks can be turned on or off via ` Menu -> View ` option. Unicode block filters only what to display contained within the range of the block. Basic Latin covers 000-007F, Latin-1 Supplement covers 0080-00FF, Latin Extended-A covers 0100-017F and so on and so forth. Global search for a glyph is affected when a certain block is currently selected. Only by selecting ` Show All ` will the global glyph search work as expected. So make it a habit of confirming that Show All is active before searching for a glyph.
 
 ### The status bar
-The status bar displays additional information describing what each menu entry and toolbar icon does. It identifies the unicode value of the glyph currently under the cursor. It shows the glyph's visual representation (if available), description and dimensions. Second only to the glyph editor window, the status bar is your next best friend on your path to becoming a font master.
+The status bar displays additional information describing what each menu entry and toolbar icon does. It identifies the unicode value of the glyph currently under the cursor. It shows the glyph's visual representation (if available), description and dimensions. The right most side displays the Unicode Block range where ` Show All ` is the default. This portion of the status bar serves also as a toggle that one can click to enable or to disables the Unicode Block display layout. Second only to the glyph editor window, the status bar is your next best friend on your path to becoming a font master.
```

Comment:
These new instructions don't belong here IMO. This section is for the status bar, which is only the the widget at the very bottom of the window. Also the Unicode block options are already documented above here.

Reply:
![image](https://user-images.githubusercontent.com/391659/209989932-fdfd47ef-5a49-4d8a-991f-2e4dff7b291d.png)
![image](https://user-images.githubusercontent.com/391659/209989963-04c175d9-7bf9-40c4-a0ee-4e41b65fd040.png)
Clicked:
![image](https://user-images.githubusercontent.com/391659/209990006-4ef3da4d-0db5-4b6e-9461-e6db147bd417.png)
Clicked again:
![image](https://user-images.githubusercontent.com/391659/209990054-fb1513e7-493c-4414-9a3a-3accb7b7364c.png)

As this feature was only recently added on FontEditor's statusbar, it only makes sense to update to document to reflect the same

Reply:
Ah gotcha. Didn't realize that was clickable!

The text is a bit confusing in that regard, maybe something like this would be easier to grok:
> The right-most segment displays the code point range of the currently selected Unicode Block. Clicking this segment will toggle the display of the Unicode Block list.